lib/elixir/lib/registry.ex

```diff
@@ -977,47 +977,47 @@ defmodule Registry do
   end
 
   @doc """
-  Counts registered keys.
+  Returns the size of the Registry (ie The number of registered keys).
```

Comment:
It should read "ie."

Reply:
No use of "ie" or similar. Please use "that is, ..." instead.